modules/reference/pages/tunable-properties.adoc

```diff
@@ -1372,6 +1372,36 @@ Maximum size of the user-space receive buffer. If `null`, this limit is not appl
 
 ---
 
+[[kafka_throughput_throttling_v2]]
+=== kafka_throughput_throttling_v2
+
+Enables an updated throttling algorithm based on a shared token bucket, introduced with Redpanda v23.3.8. Set this property to false if you need to use the quota balancing algorithm from Redpanda v23.3.7 and older.  This property defaults to true for all new or upgraded Redpanda clusters.
```

Comment:
"updated algorithm for enforcing node throughput limits..." 

(assuming this is for the node-wise throughput limits)

Reply:
> "updated algorithm for enforcing node throughput limits..."
> 
> (assuming this is for the node-wise throughput limits)

Yes, this is for controlling node-wide limit.